components/tooltip/__tests__/tooltip.test.tsx

```
@@ -590,6 +590,40 @@
+    expect(container.querySelector('.ant-tooltip')).toBeNull();
+
+    fireEvent.mouseEnter(container.querySelector('button')!);
+    await waitFakeTimer();
+
+    expect(isTooltipOpen()).toBeFalsy();
+  });
+
+  it('should work with open prop when forceRender=false', async () => {
+    const { container, rerender } = render(
+      <Tooltip title="test content" forceRender={false} open={false}>
+        <button>button</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)